checks/all modes/timing/CheckUnusedLines.cs

```diff
@@ -134,6 +142,31 @@ public override IEnumerable<Issue> GetIssues(Beatmap aBeatmap)
                     }
                 }
             }
+
+            List<TimingLine> allLines = aBeatmap.timingLines.ToList();
+
+            for (int i = 1; i < allLines.Count; ++i)
+            {
+                if (!(allLines[i] is InheritedLine currentLine)) continue;
+
+                TimingLine previousLine = allLines[i - 1];
+
+                // Check if all of the timing point's settings are the same, excluding
+                // offset and meter (meter doesn't do anything for inherited points).
+                //
+                // The SV comparison works even when the previous line is an uninherited
+                // line, because all uninherited lines' SVs are 1.
+                if (currentLine.svMult == previousLine.svMult &&
+                    currentLine.sampleset == previousLine.sampleset &&
+                    currentLine.customIndex == previousLine.customIndex &&
+                    currentLine.volume == previousLine.volume &&
+                    currentLine.kiai == previousLine.kiai &&
+                    currentLine.omitsBarLine == previousLine.omitsBarLine)
```

Comment:
Was about to say `omitsBarLine` works in a different way to other properties and isn't properly handled here, but apparently it's unused for inherited lines (just tested), so you can just remove this comparison.